common/djangoapps/student/tests/tests.py

```diff
@@ -382,6 +432,30 @@ def test_refundable_when_certificate_exists(self):
 
         self.assertFalse(enrollment.refundable())
 
+    @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+    def test_linked_in_add_to_profile_btn_with_certificate(self):
+
+        CourseModeFactory.create(
+            course_id=self.course.id,
+            mode_slug='verified',
+            mode_display_name='Verified',
+            expiration_datetime=datetime.now(pytz.UTC) - timedelta(days=1)
+        )
+
+        self.course.end = datetime.now()
+        self.course.certificates_display_behavior = 'end'
+        self.course = self.update_course(self.course, self.user.id)
+
+        GeneratedCertificateFactory.create(
+            user=self.user,
+            course_id=self.course.id,
+            status=CertificateStatuses.downloadable,
+            mode='verified'
+        )
+        response = self.client.get(reverse('dashboard'))
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
This test case should also check the rendered html for the expected URL.  One option is to use Django's `self.assertContains(response, url)`